workshop/Zadanie 1/README.md

```diff
@@ -0,0 +1,70 @@
+# Zadanie 1 - Dropdown menu
+Zadanie wykonywane z prowadzącą/prowadzącym
+
+## Jak pracować z instrukcjami?
+
+1. Dokładnie zapoznaj się z każdym krokiem zadania.
+2. Na bieżąco wprowadzaj zmiany do swojego kodu zgodnie z instrukcjami.
+3. Jeśli coś jest niejasne, pytaj śmiało!
+
+## Cel zadania
+
+Naszym celem jest sprawienie, by po najechaniu kursorem myszy na sekcję `Materiały do nauki` pojawiało się dodatkowe menu.
```

Comment:
To trzeba lepiej opisać. Jak pierwszy raz czytałem zdanie to nie rozumiałem czy to ma być na sekcji strony czy na części menu. ja bym proponował taki opis:
"Naszym celem jest sprawienie, by po najechaniu kursorem myszy na cześć menu górnego z napisem `Materiały do nauki` pojawiało się dodatkowe rozwijane menu."